source/MonoGame.Extended/Particles/Modifiers/VortexModifier.cs

```diff
@@ -20,11 +21,12 @@ public override void Update(float elapsedSeconds, ParticleBuffer.ParticleIterato
 
                 var distance2 = diff.LengthSquared();
 
-                var speedGain = _gravConst*Mass/distance2*elapsedSeconds;
+                var speedGain = _gravConst*Mass/distance2;
+                speedGain = Math.Max(Math.Min(speedGain, MaxSpeed), -MaxSpeed) * elapsedSeconds;
```

Comment:
You're basically doing a clamp here it looks like.

Couldn't this just be 
```csharp
speedGain = Math.Clamp(speedGain, -MaxSpeed, MaxSpeed) * elapsedSeconds;
```

This is available now in net8.0

Unless there is a speed performance gain by using Max and Min directly?